<launch>

	<group ns="kinect">
		
		<node pkg="kinect_driver" name="pointcloud_compressor" type="pointcloud_compressor_node" required="true" output="screen">
			<remap from="pointcloud" to="depth_camera/pointcloud_color" />
			<remap from="pointcloud_compress" to="depth_camera/pointcloud_color_compress" />
		</node>
 
	</group>
	
</launch>


 
